projects/flow.aggregation-mpi/project.py

```diff
@@ -0,0 +1,42 @@
+from flow import FlowProject, aggregator, directives
+
+
+class Project(FlowProject):
+    pass
+
+
+RANKS_PER_JOB = 2
+JOBS_PER_AGGREGATE = 8
+
+
+@Project.operation
+@directives(nranks=lambda *jobs: RANKS_PER_JOB * len(jobs))
+@aggregator.groupsof(JOBS_PER_AGGREGATE)
+def do_mpi_task(*jobs):
+    from mpi4py import MPI
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    size = comm.Get_size()
```

Comment:
```suggestion
    world_comm = MPI.COMM_WORLD
    world_rank = comm.Get_rank()
    world_size = comm.Get_size()
```
For symmetry and to disambiguate with `split_comm`, `split_rank`, and `split_size` below.